triagebot.toml

```diff
@@ -470,8 +471,8 @@ cc = ["@rust-lang/style"]
 
 [mentions."Cargo.lock"]
 message = """
-These commits modify the `Cargo.lock` file. Random changes to `Cargo.lock` can be introduced when switching branches and rebasing PRs. 
-This was probably unintentional and should be reverted before this PR is merged. 
+These commits modify the `Cargo.lock` file. Random changes to `Cargo.lock` can be introduced when switching branches and rebasing PRs.
+This was probably unintentional and should be reverted before this PR is merged.
```

Comment:
What is this change?

Reply:
Oh it's because there have been trailing spaces, and my editor removes them automatically. IIRC tidy has a lint for those but I guess it doesn't run at the top level. I can remove the changes if wanted.

Reply:
Keep this change, I was on mobile and it doesn't show the whitespaces.